presto-memory/src/test/java/com/facebook/presto/plugin/memory/TestMemoryMetadata.java

```diff
@@ -66,6 +69,37 @@ public void tableIsCreatedAfterCommits()
     }
 
     @Test
+    public void tableAlreadyExists()
+    {
+        assertThatNoTableIsCreated();
```

Comment:
Ok... but this method name already passed through @martint and @haozhun in previous PRs (in presto-blackhole and presto-memory)